GUI/CalibrateTaskWindow.xaml

```diff
@@ -71,34 +87,88 @@
+                                            </TextBox.ToolTip>
+                                        </TextBox>
                                         <Label x:Name="lblMaxPeptideLength" Content="Max Peptide Len" />
-                                        <TextBox x:Name="txtMaxPeptideLength" PreviewTextInput="PreviewIfInt" Width="45" />
+                                        <TextBox x:Name="txtMaxPeptideLength" PreviewTextInput="PreviewIfInt" Width="45" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                            <TextBox.ToolTip>
+                                                <TextBlock>
+                                                        The maximum allowed peptide length. Leave empty for no limitation. Modifications are not counted.
+                                                </TextBlock>
+                                            </TextBox.ToolTip>
+                                        </TextBox>
                                     </StackPanel>
                                     <StackPanel Orientation="Horizontal" Margin="5">
                                         <Label x:Name="MinimumScoreLabel" Content="Minimum score allowed:" />
-                                        <TextBox x:Name="minScoreAllowed" Width="45" />
+                                        <TextBox x:Name="minScoreAllowed" Width="45" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                            <TextBox.ToolTip>
+                                                <TextBlock>
+                                                    The minimum MetaMorpheus score allowed for a PSM to be reportable.
+                                                    <LineBreak/>
+                                                    Decreasing this number increases search time.
+                                                </TextBlock>
+                                            </TextBox.ToolTip>
+                                        </TextBox>
                                     </StackPanel>
                                 </StackPanel>
                                 <StackPanel Grid.Column="1">
                                     <StackPanel Orientation="Horizontal" Margin="5">
                                         <Label x:Name="initiatorMethionineLabel" Content="Initiator Methionine" />
-                                        <ComboBox x:Name="initiatorMethionineBehaviorComboBox" />
+                                        <ComboBox x:Name="initiatorMethionineBehaviorComboBox" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                            <ComboBox.ToolTip>
+                                                <TextBlock>
+                                                        Handles the Methionine presented at the beginning of most proteins (start codon).
```

Comment:
A better comment might be to state that assume cleavage of initiator methionine.